diff --git a/configure.ac b/configure.ac
index 0ab35fa3..6310c13b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -35,16 +35,16 @@ AC_CHECK_FILE([${srcdir}/src/ni/ni], [NI=${srcdir}/src/ni/ni], [
 	directory `${srcdir}/src/ni'.
 	(Try http://inform7.com/download/content/$VERSION/I7_${VERSION}_Linux_all.tar.gz)])
 ])
-AC_MSG_CHECKING([the version of ${NI}])
-AS_IF([${NI} | $GREP $VERSION > /dev/null], [AC_MSG_RESULT([$VERSION])], [
-	AC_MSG_ERROR([You have version
-	[$(${NI} | $EGREP -o '[0-9][A-Z][0-9]{2}')] of ni in src/ni. This version of Inform 7 will only work with
-	version $VERSION.
-	IMPORTANT NOTE! You must download version $VERSION of the appropriate
-	compiler package from inform7.com and put the file `ni' into the
-	directory `${srcdir}/src/ni'.
-	(Try http://inform7.com/download/content/$VERSION/I7_${VERSION}_Linux_all.tar.gz)])
-])
+#AC_MSG_CHECKING([the version of ${NI}])
+#AS_IF([${NI} | $GREP $VERSION > /dev/null], [AC_MSG_RESULT([$VERSION])], [
+#        AC_MSG_ERROR([You have version
+#        [$(${NI} | $EGREP -o '[0-9][A-Z][0-9]{2}')] of ni in src/ni. This version of Inform 7 will only work with
+#        version $VERSION.
+#        IMPORTANT NOTE! You must download version $VERSION of the appropriate
+#        compiler package from inform7.com and put the file `ni' into the
+#        directory `${srcdir}/src/ni'.
+#        (Try http://inform7.com/download/content/$VERSION/I7_${VERSION}_Linux_all.tar.gz)])
+#])
 
 ### DECLARE OPTIONS ############################################################
 
diff --git a/src/ni/ni b/src/ni/ni
new file mode 100755
index 00000000..79335f0d
Binary files /dev/null and b/src/ni/ni differ
